### DISCUSSION TITLE
defining urop interface

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#205] [hand action process; inventing 🔴]
@anupsk5053 offered help below:
based on JB's suggestion to find urop who can implement and design database and its interface, below is brief description:
based on interface for simulation https://probcomp.github.io/gen-finance/, design 
<img width="604" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/6c9bf146-297d-47e2-92c6-47234d5fc3bd">

### simulation
startup action sequence (length of 4) is stored as 4 rows. each action (part of action sequence) is mapped with financial outcome and 
<img width="962" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/f3624772-63a0-4cf8-96a2-0dcfb3b642b8">

### statistics
we can borrow animation learning 

![tsdl 161](https://github.com/Data4DM/BayesSD/assets/30194633/5ac1b53b-2a6b-4d00-9d24-755d77273075) from [autogp](https://probsys.github.io/AutoGP.jl/stable/)

|     | action | outcome1(revenue) | outcome2(paying users) |
| --- | ------ | ----------------- | ---------------------- |
| s1  | g      |                   |                        |
| s1  | ga     |                   |                        |
| s1  | gac    |                   |                        |

**Comment by anupsk5053** [comment]
Research Opportunity: Exploring Startup Success Factors

Are you curious about entrepreneurship? Wondering why some startups thrive while others falter? Is entrepreneurship merely a game of chance, or can we learn strategies to improve our odds, much like playing poker?

Our team is diving into the fascinating world of startups. We want to understand the actions that lead to success or failure. Imagine creating a compass for future entrepreneurs—a tool that guides them toward making informed decisions. But that’s not all! We’re also building a playground where founders can experiment with different scenarios before committing to a path.

Your Role: Startup Detective

As an undergraduate researcher, you’ll:

Study Startup Reports: Dive into historical startup data (we’ll provide it) and analyze their journeys.
Build a Startup Database: Document each action taken by startups and track the resulting revenue and user base (the people they serve).
Action Classification: Categorize each action into four key areas: Architecture, Technology, Customer, and Guiding Values of the founders.
Here’s a glimpse of the template you’ll use:

Skills required :
1) Basic skills in python/R
2) Data scrapping automation (not necessary but can help)
